app/controllers/flows_controller.rb

```diff
@@ -0,0 +1,147 @@
+class FlowsController < ApplicationController
+  protect_from_forgery except: :show
+
+  FLOW_TYPES = [:gyr, :ctc].freeze
+  FLOW_CONFIG = {
+    gyr: { emoji: "💵", name: "GetYourRefund Flow" },
+    ctc: { emoji: "👶", name: "CTC Flow" },
+  }
+
+  def index
+    @page_title = 'GetYourRefund Flows'
+    @flow_config = FLOW_CONFIG
+  end
+
+  def show
+    type = params[:id].to_sym
```

Comment:
You could `.map(&:to_s)` to validate before the `.to_sym`, but it's not the end of the world this way. - AL